rero_ils/modules/items/api/record.py

```diff
@@ -355,6 +355,18 @@ def holding_circulation_category_pid(self):
                     self.holding_pid).circulation_category_pid
         return circulation_category_pid
 
+    @property
+    def issue_inherited_first_call_number(self):
+        """Get issue inherited first call number.
+
+        For item of type issue, when the issue first call number is missing,
+        it returns the parent holdings first call number. if exists.
```

Comment:
Suggestion: ` it returns the parent holdings first call number if exists.` (remove `.` after number)